File: source/AndroidClient/AtMiles/app/src/main/java/no/miles/atmiles/employee/EmployeeDetailsResponse.java
package no.miles.atmiles.employee;

public class EmployeeDetailsResponse {
    public String Id;
    public String GlobalId;
    public String CompanyId;
    public String OfficeName;
    public String Name;
    public String DateOfBirth;
    public String JobTitle;
    public String PhoneNumber;
    public String Email;
    public Address PrivateAddress;
    public String Thumb;
    public Tag[] Competency;
    public BusyTime[] BusyTimeEntries;
    public String[] KeyQualifications;
    public Description[] Descriptions;
    public double Score;

    public static class Tag{
        public String Category;
        public String Competency;
        public String InternationalCompentency;
        public String InternationalCategory;
    }

    public static class BusyTime
    {
        public String Id;
        public String Start;
        public String End;
        public int PercentageOccupied;
        public String Comment;
    }

    public static class Description
    {
        public String InternationalDescription;
        public String LocalDescription;
    }

    public static class Address {
        public String Street;
        public String PostalCode;
        public String PostalName;
    }
}

File: source/Backend/Api/Models/Api/Employee/EmployeeDetailsResponse.cs
using System;
using System.Collections.Generic;

namespace no.miles.at.Backend.Api.Models.Api.Employee
{
    public class EmployeeDetailsResponse
    {
        public string Id;
        public string GlobalId;
        public string CompanyId;
        public string OfficeName;
        public string Name;
        public DateTime DateOfBirth;
        public string JobTitle;
        public string PhoneNumber;
        public string Email;
        public Address PrivateAddress;
        public string Thumb;

        public Tag[] Competency;

        public List<BusyTime> BusyTimeEntries;

        public List<string> KeyQualifications;

        public List<Description> Descriptions;

        public double Score;

        public class BusyTime
        {
            public string Id;
            public DateTime Start;
            public DateTime? End;
            public short PercentageOccupied;
            public string Comment;
        }

        public class Description
        {
            public string InternationalDescription;
            public string LocalDescription;
        }

        public class Address
        {
            public string Street;
            public string PostalCode;
            public string PostalName;
        }

        public class Tag
        {
            public string Category;
            public string Competency;
            public string InternationalCompentency;
            public string InternationalCategory;
        }

        public string FirstName { get; set; }
        public string LastName { get; set; }
    }
}